src/pages/admin/create-blog.tsx

```diff
@@ -0,0 +1,643 @@
+          dateFormat="yyyy-MM-dd"
+          minDate={new Date()}
+        /><br/>
+
+        <b className="calendar-label">{this.state.disableCalendar ? "No expiry" : "Select expiry date"}</b>
+        <button onClick={()=>this.setState({ disableCalendar: !this.state.disableCalendar })}>None</button>
+        <DatePicker 
+          selected={this.state.expireDate} 
+          onChange={this.handleExpireDate} 
+          dateFormat="yyyy-MM-dd"
+          disabled={this.state.disableCalendar}
+          minDate={new Date()}
+        /><br/>
+
+        <label>
+          Delete a blog (type in id):
+          <input type="text" value={this.state.delete} onChange={(event:any) => this.setState({ delete: event.target.value})} />
+        </label>
+        <label>
+          Type "The admins have warned me", then click to confirm:
+          <input type="text" value={this.state.understand} onChange={(event:any) => this.setState({ understand: event.target.value})} />
+        </label>
+        <button className="tags-button" style={{background: "red"}} onClick={this.handleDeleteBlogPost}>DELETE</button>
+
+      </div>
+    )
+  }
+
+  renderTextInput() {
+    return (
+      <div className="editor-container">
+        <label>
+          Title:
+          <input className="small-input" type="text" value={this.state.title} onChange={(event:any)=> {this.setState({ title: event.target.value, editMode: false, selectedBlogSeries: [] }); this.updateBlogField('blogTitle', event.target.value)} } />
+        </label>
+
+        <label>
+          Author:
+          <input className="small-input" type="text" value={this.state.author} onChange={(event:any)=> {this.setState({ author: event.target.value, editMode: false, selectedBlogSeries: [] }); this.updateBlogField('author', event.target.value)} } />
+        </label>
+
+        <label style={this.state.desc.length >= 180 ? {color: "red"} : {color: "black"}}>
+          Description ({this.state.desc.length + " of 210 characters"}):
+          <textarea className="big-input" maxLength={210} value={this.state.desc} onChange={(event:any)=> this.setState({ desc: event.target.value})} />
+        </label>
+
+        <Editor
+          editorState={this.state.editorState}
+          onEditorStateChange={this.onChange}
+          spellCheck={true}
+          toolbar={{
+            options: ['inline', 'blockType', 'fontSize', 'list', 'link', 'emoji', 'image', 'history'],
+            inline: {
+              options: ['bold', 'italic', 'underline', 'strikethrough', 'superscript', 'subscript'],
+            },
+            list: {
+              options: ['unordered', 'ordered']
+            },
+            image: {
+              uploadEnabled: true,
+              uploadCallback: async (file: any) => {
+                  const filepath = "/bloguploads/" + uuidv1() + file.name;
+                  await Storage.put(filepath, file, {
+                      contentType: "image/*"
```

Comment:
acl: 'public-read' I believe should do it....

Reply:
Didn't to the trick unfortunately.